lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -887,28 +951,28 @@ impl X64FunctionCode {
             .insert(m.state.wasm_inst_offset, SuspendOffset::Trappable(offset));
     }
 
-    #[allow(dead_code)]
-    fn mark_inline_breakpoint(
-        a: &mut Assembler,
-        m: &Machine,
-        fsm: &mut FunctionStateMap,
-        control_stack: &mut [ControlFrame],
-    ) {
-        let state_diff_id = Self::get_state_diff(m, fsm, control_stack);
-        let offset = a.get_offset().0;
-        fsm.trappable_offsets.insert(
-            offset,
-            OffsetInfo {
-                end_offset: offset
-                    + get_inline_breakpoint_size(ARCH, Backend::Singlepass)
-                        .expect("cannot get inline breakpoint size"),
-                activate_offset: offset,
-                diff_id: state_diff_id,
-            },
-        );
-        fsm.wasm_offset_to_target_offset
-            .insert(m.state.wasm_inst_offset, SuspendOffset::Trappable(offset));
-    }
+    // #[allow(dead_code)]
```

Comment:
Probably just remove this? Either leave it as-is (marked allow(dead_code)) or remove it entirely.